flocker/acceptance/obsolete/__init__.py

```diff
@@ -0,0 +1,6 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
```

Comment:
One day, in the distant future, I'll remember that we're not called Hybrid Logic Ltd anymore. Old habits.... :smile: